ci_framework/roles/set_openstack_containers/README.md

```diff
@@ -12,10 +17,72 @@ used during OpenStack services deployment using meta operator.
 * `cifmw_set_openstack_containers_overrides`: Extra container overrides. Defaults to `{}`
 
 ## Examples
+
+Notes: Make sure we installed the operator before updating the environment variable
```

Comment:
`the operator before`

which operator be explicit. do you mean generally all operators?

`Make sure operators are installed before updating the relevant environment variable(s)` 

?

Reply:
Thank you @marios . I updated the docs. It refers to meta operator.